docs/docs/playtesting/testing-in-app.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 1
+---
+
+# Testing in Rune App
+
+Rune lets you easily playtest your game in the Rune app before it's published to everyone. This is a great way to test the actual multiplayer experience of your game with friends and [team members](../publishing/collaboration#team-roles) before releasing it to the Rune community.
+
+Upload your game by running: 
+
+```sh
+npx rune-games-cli@latest create
+```
+After this, you'll see your game in the app even though it's not publicly available yet. Here's [more details on uploading your game](../publishing/publishing-your-game.md) if interested.
+
+Then create a room in the app and select your game in the list. You can also invite others who are not on your team to playtest with you. They can't see your game in the list, but can play the game with you once you've selected it.
+
+## Playing New Versions before Release
+
+Once your game has been published, everyone in the Rune community will have access to the latest active version of the game.
+
+You're likely continuously improving your game and anyone on your team will have access to the latest uploaded draft / in review version. In this way, you are able to playtest the actual multiplayer game experience of new versions before they're released!
```

Comment:
```suggestion
To support game development after it is released to everyone, your team members will have access to the latest draft / in review version instead of the published one. This way you are able to playtest the latest game changes in the app before they're released to everyone!
```